pages/api/prescriptions/create.ts

```diff
@@ -47,6 +48,40 @@ export default async function handler(
         }
       })
 
+      const patient = await prisma.patient.findUnique({
+        where: { id: patientId }
+      })
+
+      const user = await prisma.user.findUnique({
+        where: { id: patient?.userId }
+      })
+
+      const location = await prisma.location.findUnique({
+        where: { id: locationId }
+      })
+
+      if (!user?.phoneNumber) {
+        res.status(400).json({
+          message:
+            "User does not have a phone number. Please add a phone number to this users's account"
+        })
+      }
+
+      let phone_number = user?.phoneNumber
+
+      if (phone_number?.charAt(0) === '1') {
+        phone_number = '+' + phone_number
+      } else {
+        phone_number = '+1' + phone_number
```

Comment:
is there a reason why we are using snake case here? generally Typescript is `camelCase`